R/create.ccf.heatmap.R

```diff
@@ -5,6 +5,7 @@ create.ccf.heatmap <- function(
     distance.method = 'euclidean',
     xaxis.lab = '',
     xlab.label = 'SNVs',
+    yaxis.lab = colnames(x),
```

Comment:
If I'm understanding correctly, we can't use the result of `colnames` as a default argument here. This should be `yaxis.lab = NULL`, then set the default inside the function. Something like:

```
if (is.null(yaxis.lab)) {
    yaxis.lab <- colnames(x);
    }
```